django2_codemods/auth_anonymous_update.py

```diff
@@ -0,0 +1,36 @@
+import sys
+
+from typing import Optional
+from bowler import Query, LN, Capture, Filename, TOKEN, SYMBOL
+from fissix.pytree import Node, Leaf
+
+
+# usage of the User.is_authenticated() and User.is_anonymous() methods is deprecated. Remove the last parenthesis.
+
+
+def remove_parenthesis(node: LN, capture: Capture, filename: Filename) -> Optional[LN]:
```

Comment:
Minor spelling/grammar note: the plural "parentheses" is more accurate than the singular "parenthesis" here since you're removing two such symbols.